langchain-llamaindex-slackbot/src/requirements.txt

```diff
@@ -2,7 +2,7 @@ langchain==0.0.125
 openai==0.27.2
 slack-bolt==1.16.2
 slack-sdk==3.20.0
--e  git+https://github.com/zenml-io/zenml.git@develop#egg=zenml
+zenml==0.40.0
```

Comment:
Does this work? I tried with 0.39.1 but it didnt work due to some internal error

Reply:
Unsure. Will have to wait for 0.40.1 to test. This is just a draft for now.

Reply:
It works.